x/incentives/keeper/distribute.go

```diff
@@ -21,9 +21,14 @@ func (k Keeper) getDistributedCoinsFromGauges(gauges []types.Gauge) sdk.Coins {
 }
 
 func (k Keeper) getToDistributeCoinsFromGauges(gauges []types.Gauge) sdk.Coins {
-	// TODO: Consider optimizing this in the future to only require one iteration over all gauges.
-	coins := k.getCoinsFromGauges(gauges)
```

Comment:
`func (k Keeper) getCoinsFromGauges(gauges []types.Gauge) sdk.Coins {..}` is unused now, maybe you can remove it

Reply:
thanks, I'll remove it